cassiopeia/core/summoner.py

```diff
@@ -209,10 +209,3 @@ def verification_string(self) -> str:
         from .thirdpartycode import VerificationString
         vs = VerificationString(summoner=self, region=self.region)
         return vs.string
-
-    @lazy_property
-    def ranks(self):
-        ranks = {}
-        for position in self.league_positions:
```

Comment:
We should be able to keep this functionality by replacing `self.league_positions` will ~ `self.league_entries`.

Reply:
Yup, I don't know how I missed that as I even was looking for alternatives. I have re-added it using league_entries and a unit test to ensure it returns the correct information